Dockerfile

```diff
@@ -20,33 +20,26 @@
 #  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 #  THE SOFTWARE.
 
-FROM maven:3.5.2-jdk-8
+FROM maven:3.5.2-jdk-8 as builder
 LABEL Description="Base image for running Jenkins Plugin Compat Tester (PCT) against custom plugins and Jenkins cores" Vendor="Jenkins project"
```

Comment:
Maybe this label needs to be moved from the builder to the final distributed image?

Reply:
yes, you are right